src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -298,12 +299,22 @@ export const TradeInput = ({ isCompact, tradeInputRef, onChangeTab }: TradeInput
     [dispatch],
   )
 
+  const assetFilterPredicate = useCallback((asset: Asset) => {
+    return asset.chainId !== KnownChainIds.SolanaMainnet
+  }, [])
+
+  const chainIdFilterPredicate = useCallback((chainId: ChainId) => {
+    return chainId !== KnownChainIds.SolanaMainnet
+  }, [])
+
```

Comment:
q: bit of an eng/product overlap on this one but what's the rationale behind this guy? 

Just because we have no support for Solana tokens (which we actually do for USDC under Chainflip flag in develop, and will have for others in https://github.com/shapeshift/web/pull/8120), doesn't mean we should add monkey patches to hide Solana IMO.

There are two issues with doing this, one architectural, the other which is purely a product opinion of mine:

- This makes removing the conditions when going live painful, since we have to find the exact location of monkey patches. I definitely lost a few hours trying to find these and even then, missed some instances (which were thankfully caught by @NeOMakinG 🐐)
- From a conceptual standpoint, I think asset list/search as well as chain selection/related assets list shouldn't know about the current WIP for said chain - if no route/supported swapper is found for user selection, then this is handled gracefully as we return an empty state
<img width="1052" alt="image" src="https://github.com/user-attachments/assets/883572e3-64fa-4d2c-af84-cf7c1bf88800">

Reply:
Also spotted that Solana assets are now hidden as available assets too in develop, not only as related assets, which is also a bug:

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/f6f8e48d-98c1-4015-bfc5-f7f0bb25b987">

Reply:
Sounds good, I've removed solana filtering from spot 72d6224b46d92a8f470ddadb4dc6f109b7fe0e5b